num2words/lang_SL.py

```diff
@@ -16,8 +16,6 @@
 # Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
 # MA 02110-1301 USA
 
-# -*- coding: utf-8 -*-
```

Comment:
Why is this removed here but added everywhere else?

Reply:
It's already at the top of the file.

Reply:
I made a mistake during my rebase, I will fix it later.